rollup.config.js

```diff
@@ -1,27 +1,26 @@
-'use strict';
-
-var babel = require('rollup-plugin-babel');
+import babel from '@rollup/plugin-babel';
 // uglify handles only es5 code, so this also acts as smoke test against shipping es2015+ syntax
-var uglify = require('rollup-plugin-uglify').uglify;
-var pkg = require('./package.json');
+import { uglify } from "rollup-plugin-uglify";
+import pkg from './package.json';
 
-var banner = '//  Ramda v' + pkg.version + '\n'
+const banner = '//  Ramda v' + pkg.version + '\n'
   + '//  https://github.com/ramda/ramda\n'
   + '//  (c) 2013-' + new Date().getFullYear() + ' Scott Sauyet, Michael Hurley, and David Chambers\n'
   + '//  Ramda may be freely distributed under the MIT license.\n';
 
-var input = 'source/index.js';
-
-var config = {
-  input: input,
+const config = {
+  input: 'source/index.js',
   output: {
     format: 'umd',
     name: 'R',
     exports: 'named',
     banner: banner
   },
   plugins: [
-    babel({ presets: [['@babel/preset-env', { targets: { ie: '11' } }]]})
+    babel({
+      babelHelpers: 'bundled',
```

Comment:
This is recommended when using the newer version of this plugin: https://www.npmjs.com/package/@rollup/plugin-babel#babelhelpers

As no babel helpers are used by ramda, this has no effect.